jdat_notebooks/README.md

```diff
@@ -8,11 +8,11 @@ that can be "live" on the spacetelescope notebooks repository.
 These notebooks can have many varied science cases, but follow a relatively
 standard workflow:
 
-1. Notebook Concept
-2. Notebook Draft
+1. Draft Stage
+2. Baseline Stage
 3. Notebook-driven development
-4. Integrated Notebook
-5. Final/Public Notebook
+4. Advanced Stage
```

Comment:
```suggestion
4. Advanced Integration Stage
```

I know we want to follow the DAWG wording, but I'm thinking we still want *something* that indicates this stage is really about "having the tools and the notebooks work together" instead of just "better" (which is what "advanced" sounds like to me).

I'm not necessarily attached to this *exact* wording but hopefully the idea gets across here.

Reply:
Sounds good to me!